Credits page + Misc adjustments

keijeizei opened the pull request:
- [x] Create basic credits layout
- [x] Finish Credits page (just waiting for the other messages from other projects)

Misc:
- [x] Add Coco's picture at the title
- [x] Make fonts with `em` units responsive using breakpoints in `globals.scss` Fixes #74 
- [x] Fix gap sizes Fixes #92 
- [x] Add /fanart in footer

keijeizei committed:
Create credits layout

keijeizei committed:
Make fonts responsive

keijeizei committed:
Remove dummy text

keijeizei committed:
Add Coco picture in title

keijeizei committed:
Replace Coco picture with higher quality

keijeizei committed:
Change footer font size to em units

keijeizei committed:
Add partial list of developers

keijeizei committed:
Fix credits page responsiveness

keijeizei committed:
Merge main

keijeizei committed:
Fix videoboard merge conflicts

keijeizei committed:
Merge branch 'Kaichou-Project:main' into credits

keijeizei committed:
Add special thanks in credits

keijeizei committed:
Merge branch 'credits' of https://github.com/keijeizei/kaichou-graduation into credits

keijeizei committed:
Add more names

keijeizei committed:
Add /fanart in footer

keijeizei committed:
Adjust ToTheTop font size

keijeizei committed:
Remove green outline from Coco picture

keijeizei committed:
Fix conflicts

keijeizei committed:
Improve videoBoard css

keijeizei committed:
Update developers name

keijeizei committed:
Hide Coco picture when submitting

RedMaps commented on packages/frontend/components/Navigation/Navigation.tsx:
```diff
@@ -95,7 +95,12 @@ export default function Navigation({
       </div>
 
-      {mode != Mode.FORM && <SakuraParticles />}
+      {mode != Mode.FORM && (
```
```suggestion
      {mode !== Mode.FORM && (
```

RedMaps commented on packages/frontend/pages/credits/index.tsx:
```diff
@@ -1,12 +1,96 @@
+        <br />
+        <br /> Thank you for all the smiles and laughter you&apos;ve given us
+        this past year and a half. We appreciate all the effort you put in to
+        innovate the world of vtubers and paving the way for HoloEN. We grateful
```
```suggestion
        innovate the world of vtubers and paving the way for HoloEN. We&apos;re grateful
```

RedMaps reviewed (changes requested):
Job well done, some minor issues and I'd advocate for giving the credits page a big padding on the left and right so the text is easier to read and the visuals are improved, maybe around this much:
<img width="1888" alt="Screenshot 2021-07-01 at 11 21 50" src="https://user-images.githubusercontent.com/16868419/124100310-aa11b400-da5e-11eb-975c-7d1d05054805.png">
Edit: I feel like the developers section should be a list from top to bottom... It kind of looks confusing to read in this horizontal way

keijeizei committed:
Address requested changes

keijeizei committed:
Update developers name

azuraga committed:
update credits

keijeizei committed:
Add more descriptions in credits

nyxordinal commented:
The page is already good, one comment if I may, could you increase the font size? 
I think it's a bit difficult for eyes to read the text in current font size

keijeizei committed:
Fix font sizes

Increase credits font size and conform the Homeboard font sizes to the global font sizes in globals.scss

keijeizei committed:
Increase font size of loading text

nyxordinal reviewed (approved):
Nice work !
I have no comments on this PR, if @RedMaps also has no comment then we can proceed to merge this PR

keijeizei committed:
Center videos on small screens

RedMaps reviewed (approved):
Looks good now!
